.circleci/config.yml

```diff
@@ -71,6 +71,16 @@ references:
         sudo apt-get --assume-yes install python3-pip
         sudo pip3 install awscli
 
+  install_yq: &install_yq
+    run:
+      name: Install yq (YAML parser)
+      command: |
+        wget https://github.com/mikefarah/yq/releases/download/${VERSION}/${BINARY}.tar.gz -O - |\
+        tar xz && mv ${BINARY} /usr/bin/yq
+      environment:
+        VERSION: v4.6.3
+        BINARY: yq_linux_amd64
+
```

Comment:
I feel that one of these 2: 
    Use sudo add-apt-repository ppa:rmescandon/yq -y && sudo apt update && sudo apt install yq -y to install from Ubuntu PPA.
    Use snap install yq to install using Snappy.

might be better than grabbing directly from github from a security perspective